automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookCustomizationSpec.scala

```diff
@@ -106,5 +106,29 @@ final class NotebookCustomizationSpec extends GPAllocFixtureSpec with ParallelTe
         }
       }
     }
+
+    "should populate user-specified environment variables" in  { billingProject =>
+      implicit val ronToken: AuthToken = ronAuthToken
+
+
+      withNewCluster(billingProject, request = defaultClusterRequest.copy(customClusterEnvironmentVariables = Map("KEY" -> "value"))) { cluster =>
+        withWebDriver { implicit driver =>
+          withNewNotebook(cluster, Python3) { notebookPage =>
+            notebookPage.executeCell("import os")
+            val envVar = notebookPage.executeCell("os.getenv('KEY')")
+            envVar shouldBe Some("'value'")
+          }
+        }
+      }
+
+      withNewCluster(billingProject, request = defaultClusterRequest.copy(customClusterEnvironmentVariables = Map("KEY" -> "value"))) { cluster =>
```

Comment:
Could this use the same cluster as above? It's costly to create a lot of clusters in tests; better to reuse them if possible.
```
      withNewCluster(billingProject, request = defaultClusterRequest.copy(customClusterEnvironmentVariables = Map("KEY" -> "value"))) { cluster =>
        withWebDriver { implicit driver =>
          withNewNotebook(cluster, Python3) { notebookPage =>
            notebookPage.executeCell("import os")
            val envVar = notebookPage.executeCell("os.getenv('KEY')")
            envVar shouldBe Some("'value'")
          }
          withNewNotebook(cluster, RKernel) { notebookPage =>
            val envVar = notebookPage.executeCell("Sys.getenv('KEY')")
            envVar shouldBe Some("'value'")
          }
        }
      }
```

Reply:
Yes, can do